### DISCUSSION TITLE
How to create a media player in RDR2

### DISCUSSION THREAD

**Discussion by scott-cole** [kibook/pmms#39] [Q&A]
Hi i have used /pmms_add but im not sure what i am needing to do to make the theatre screen in blackwater a media player or a phonograph, dumb question i know, any help pleasE :)

**Comment by kibook** [comment]
I don't believe RedM supports DUI (yet), so you aren't able to display stuff on a texture like in FiveM.

On my RedM server, I have the following setup for the Blackwater theatre:

```lua
Config.defaultMediaPlayers = {
        {
                label = "Imperial Theatre",
                position = vector3(-773.026, -1362.673, 43.22),
                spawn = true,
                rotation = vector3(0, 0, -90),
                invisible = true,
        },
}
```

What this does is automatically spawn an invisible phonograph just below the screen. Playing a video on it with the video checkbox checked will display the video on a "screen" "above" the phonograph (like in the [example video](https://imgur.com/2jRYlSe)), and therefore it kind of looks like it's playing on the theatre screen.
